content/root/quick-start-guide.textile

```diff
@@ -42,6 +42,10 @@ p(tip#terms). The terms used in the guide such as 'connection', 'channel', 'mess
 
 You're currently viewing the guide in <span lang="javascript">JavaScript</span><span lang="nodejs">Node.js</span><span lang="java">Java</span><span lang="ruby">Ruby</span><span lang="objc">Objective-C</span><span lang="python">Python</span><span lang="php">PHP</span><span lang="swift">Swift</span><span lang="csharp">C# .NET</span><span lang="flutter">Flutter</span><span lang="go">Go</span>. If you would like to view it in another language, use the selector above to change it. 
 
+h2(#step-0). Create an Ably account
+
+The following code sample uses a demo API key to authenticate however you can use your own API key if you already have an account, or "sign up":https://ably.com/signup for a free account to obtain one. Just ensure that the key provides the subscribe and publish "capabilities":/core-features/authentication#capabilities-explained.
```

Comment:
I think this sentence is missing some punctuation. Perhaps "The following code sample uses a demo API key to authenticate; however, you can ..."

Reply:
Agreed 👍 And I'd also like to move the sign-up CTA into the first sentence. I'm awaiting Ably GitHub access and then I'll create a new PR with a few updates before this goes live.